source/includes/limitations/4.3.rst

```diff
@@ -76,8 +76,13 @@ in the MongoDB manual.
 Mutual Recursion
 ~~~~~~~~~~~~~~~~
 
-You cannot specify a **mutually recursive** type as a type parameter in version
-4.3 of the driver.
+.. important::
+
+   In this driver version, you cannot specify a mutually recursive type as a
```

Comment:
Suggestion:
This seems like repeated content with the first sentence. I think it could be more straightforward to lead this section with the sentence "You cannot specify a ..." and then have an informational admonition that follows mentioning that you can use mutually recursive types in type parameters in v4.11 and later.

Reply:
Agreed that would make things read a bit smoother. Since the other versions (v4-10) have this admonition I'm leaning towards merging as-is for consistency's sake, but I can create a quick cleanup ticket to reword at a later time